### PR TITLE
Scaling up

### PR DESCRIPTION
**Pull request by JulienLamour**

### CODE REVIEW

**Comment by serbinsh** [comment]
@JulienLamour are you going to request code review on this PR before merging?  Its rather large so it might be a good idea.  You could request me and @neo0351 or even @DavidsonKen but we will just need to pull down the branch to test first

**Comment by JulienLamour** [comment]
Actually I modified of a little bit of everything so it is better to merge it. But that's a good idea to do a review. There is currently no vignette for the scaling up, so maybe I should do one before a review! I understood better how works the .md files so I also have to move the documentation I did for Angie on one of the folders. I can do that today and let you then do the review.

File: DESCRIPTION [hunk 1]
```diff
@@ -11,8 +11,19 @@ License: What license it uses
...
+    bbmle,
+    readxl,
+    stringr,
+    tidyverse,
+    readr,
+    tealeaves,
+    reshape2,
+    ggplot2,
+    BioCro
+Remotes: ebimodeling/biocro
```
Comment (serbinsh) [inline DESCRIPTION:26]: Nice

**Comment by serbinsh** [comment]
@JulienLamour Nice work on this PR!  but you merge dont forget to build the docs using roxygen2

**Pull request merged**